index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+        rel="stylesheet"
+        href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+        integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+        crossorigin="anonymous"
+        />
+    <title>Teribute page</title>
+</head>
+<body>
+<main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
```

Comment:
#h1
remove id and use only h1 because it is the only h1 you have

Reply:
h1 is a tag and does not require an id because it is the only h1. this will be easy for you to code and also avoid complication